.github/workflows/ci.yml

```
@@ -163,3 +163,23 @@ jobs:
       with:
         name: ${{ matrix.browser-channel }}-${{ matrix.os }}
         path: pw-log.txt
+
+  build-conda:
+    name: Conda Build
+    strategy:
+      fail-fast: false
+      matrix:
+        os: [ubuntu-latest, windows-latest, macos-latest]
+    runs-on: ${{ matrix.os }}
+    steps:
+      - uses: actions/checkout@v2
+        with:
+          fetch-depth: 0
+      - name: Get conda
+        uses: conda-incubator/setup-miniconda@v2
+        with:
+          channels: conda-forge
+      - name: Prepare
+        run: conda install conda-build conda-verify
+      - name: Build
+        run: conda build .
```

```suggestion
        run: conda build --user microsoft .
```
not sure if this needed but I used it locally. Since a token is for a user we need to _somehow_ explicitly say that it should use the microsoft group.